src/dotty/tools/dotc/repl/CombinedClassLoader.scala

```diff
@@ -0,0 +1,37 @@
+package dotty.tools
+package dotc
+package repl
+
+import scala.util.Try
+
+class CombinedClassLoader(loaders: List[ClassLoader]) extends ClassLoader {
```

Comment:
Yes, this guy needs to go away - not sure how yet though. Every other solution seems to create a circular dependency on sbt 👎